app/assets/stylesheets/comp-vue-beeswarm.scss

```diff
@@ -1,9 +1,8 @@
 :root {
   --grey-line: #c4c4c4;
-  --blue-text: #0072bc;
   --white: #fff;
   --black: #000;
-  --grey: #ccc;
+  --grey-border: #ccc;
```

Comment:
Try to be consequent when you name variables. You know that `#ccc` is gonna be used in a border, but `#ccc` is NOT a border, it's a color.

What if I have multiple colors? You may use the common schema: `--light-grey; --lighest-grey; --dark-grey; //etc...` or a better approach when there's a few props: `--border-color-grey;`
_grey-line_, _black_, _white_ could also be renamed as simply `--$COLOR` or `--color-$COLOR`, depending on how many custom vars you have.

Reply:
Done!